index.js

```diff
@@ -32,9 +40,14 @@ getFullNames(usersArray);
 
 const getUsersCreditDetails = arr => {
   // Your code goes here ...
-};
-
+  const usersCreditDetails = [];
+    for (const {firstName: n, lastName: g, balance: j} of usersArray) {
```

Comment:
This might be overly complex to read. Its correct, but might be better, in readability in two weeks to get a variable, and directly below, you can destructure the variable